ml_tools/eolearn/ml_tools/train_split.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tasks used for train set preparation.
+"""
+import numpy as np
+
+from eolearn.core import EOTask
+
+
+class TrainSplitTask(EOTask):
```

Comment:
In ML community the methods that split samples in two or more independent sets usually have`train_test` in their name. I would suggest to follow this nomenclature and suggest to rename the task to `TrainTestSplitTask`.

Reply:
Done.